content/documentation/overview.md

```diff
@@ -8,12 +8,77 @@ heroImage: "img/banners/banner2.png"
 submenus: documentation
 ---
 
-# Overview of the QGIS Documentation Resources
+{{< image-bar
+    "For users:/documentation/docperson.png"
```

Comment:
Probably need another note in the README, we should use simple urls i.e.docperson.png rather than the full path, but I might need to tweak the image-bar a bit. so leave it as is for now and I will tweak.